client/gutenberg/extensions/publicize/settings-button.jsx

```diff
@@ -25,15 +25,28 @@ import { Component } from '@wordpress/element';
  * Internal dependencies
  */
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import getSiteFragment from 'gutenberg/extensions/presets/jetpack/editor-shared/get-site-fragment';
 
 class PublicizeSettingsButton extends Component {
+	getButtonLink() {
+		const siteFragment = getSiteFragment();
+
+		// If running in WP.com wp-admin or in Calypso, we redirect to Calypso sharing settings.
+		if ( siteFragment ) {
+			return `https://wordpress.com/sharing/${ siteFragment }`;
```

Comment:
Is it okay that this won't bring you to `calypso.localhost:3000/sharing` or `horizon.wordpress.com/sharing` etc when in Calypso using those environments?

Reply:
Should be okay for now - it opens in a new window anyway 👍 We could easily figure a fix for it later.